run_qemu.sh

```diff
@@ -144,6 +144,21 @@ distro_vars="${script_dir}/${_distro}_vars.sh"
 
 pushd "$_arg_working_dir" > /dev/null || fail "couldn't cd to $_arg_working_dir"
 
+# save the working directory
+working_dir=$(pwd)
```

Comment:
Why save it again? It's already in `"$_arg_working_dir"`

Reply:
pwd guarentees we have that relative directory after doing the push where ever _arg_working_dir was.

FWIW AFAIK no one uses _arg_working_dir.  So it is probably almost untested.  But this seemed safe if it does work.

Reply:
Ah yes of course. Then please:

```suggestion
abs_working_dir=$(pwd)
```

Please also drop or change the comment, it threw me off.

Reply:
ok sure!